src/api/spec/features/webui/image_templates_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "browser_helper"
+# WARNING: This test require real backend answers for projects/packages, make
+# sure you uncomment this line and start a test backend.
+# CONFIG['global_write_through'] = true
+
+RSpec.feature "ImageTemplates", type: :feature, js: true do
+  let(:admin) { create(:admin_user, login: 'admin') }
+  let(:project) { create(:project, name: "my_project") }
+  let!(:attrib) { create(:template_attrib, project: project) }
+  let!(:package1) { create(:package, project: project, name: "first_package") }
+  let!(:package2) { create(:package, project: project, name: "second_package") }
+
+  scenario "branch image template" do
+    login(admin)
```

Comment:
yes! at the moment only admins can see the link in the root_path, we can remove it when this change.